google/cloud/completion_queue.h

```diff
@@ -208,6 +208,32 @@ class CompletionQueue {
         absl::make_unique<Wrapper>(impl_, std::forward<Functor>(functor)));
   }
 
+  /**
+   * Asynchronously run a functor on a thread `Run()`ning the `CompletionQueue`.
+   *
+   * @tparam Functor the functor to call on the CompletionQueue thread.
+   *   It must satisfy the `void()` signature.
+   * @param functor the value of the functor.
+   * @return an asynchronous operation wrapping the functor; it can be used for
+   *   cancelling but it makes little sense given that it will be completed
+   *   straight away
```

Comment:
s/$/./

Reply:
Actually the whole `@return` section (here and in the other overload) was wrong, removed.